spec/rubocop/cop/rspec/repeated_example_group_body_spec.rb

```diff
@@ -226,6 +226,50 @@
     RUBY
   end
 
+  it 'skips `skip` and `pending` statements with arguments' do
+    expect_no_offenses(<<-RUBY)
+      context 'processed' do
+        skip 'some reason'
+      end
+
+      context 'processed' do
+        skip 'some reason'
+      end
+
+      context 'processed' do
+        pending 'another reason'
+      end
+
+      context 'processed' do
+        pending 'another reason'
+      end
+    RUBY
+  end
+
+  it 'registers offense correctly if `skip` and `pending` have block' do
+    expect_offense(<<-RUBY)
+      describe 'repeated' do
+      ^^^^^^^^^^^^^^^^^^^^^^ Repeated describe block body on line(s) [5]
+        skip { blah }
+      end
+
+      describe 'this is repeated' do
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Repeated describe block body on line(s) [1]
+        skip { blah }
+      end
+
+      describe 'repeated too' do
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^ Repeated describe block body on line(s) [13]
+        pending { blah }
+      end
+
+      describe 'this is repeated too' do
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Repeated describe block body on line(s) [9]
+        pending { blah }
```

Comment:
What if several skipped examples only contain a comment, e.g.
```ruby
# FIXME: don't forget to write a spec for this
```
?

Reply:
Comments are just ignored on compare due they are not in AST.
This PR didn't target empty example blocks.